webrecorder/webrecorder/rec/main.py

```diff
@@ -1,12 +1,15 @@
 from gevent import monkey; monkey.patch_all()
 
+import os
+from ast import literal_eval
+
 from webrecorder.utils import load_wr_config, init_logging, spawn_once
 from webrecorder.rec.webrecrecorder import WebRecRecorder
 
 
 # =============================================================================
 def init():
-    init_logging(debug=True)
+    init_logging(debug=literal_eval(os.environ.get('WR_DEBUG', 'True')))
```

Comment:
Very small tweak: can this use the `get_bool` from webrecorder.utils instead?

```suggestion
    init_logging(debug=get_bool(os.environ.get('WR_DEBUG', 'True')))
```